src/components/20-molecules/datepicker/index.js

```diff
@@ -729,6 +748,63 @@ class AXADatepicker extends NoShadowDOM {
     // any preexisting error should be cleared
     this.error = null;
   }
+
+  // Switches to the next allowed month, when 'direction' is positive.
+  // Switches to the previous allowed month, when 'direction' is negative.
+  // Has no effect, when there is no next/previous allowed month.
+  showNextMonth(direction = 0) {
+    const MIN = 0;
+    const MAX = 11;
+
+    let month = parseInt(this.month, 10);
+    let year = parseInt(this.year, 10);
+
+    if (direction < 0) {
+      if (month !== MIN) {
+        month -= 1;
+      } else {
+        const prevYear = this.getNextYear(-1);
```

Comment:
getNextYear is also misnamed and can also uniformly take monthOffset as argument.